examples/browser-sdk/require.js

```diff
@@ -0,0 +1,1115 @@
+var requirejs, require, define;
```

Comment:
Did you write this? :hushed: Or is this just requireJS? Do we want to include the requireJS script here, or write up a rollup config?

Reply:
Oh sorry, no, this is just literally require.js. The HTML file is based on the [original browser SDK sample](https://github.com/AssemblyScript/examples/blob/master/sdk/index.html), but it loaded RequireJS from a CDN and I wanted the sample to work offline. So yeah, I got nothin’ todo with PlayStation 3 😂